package.json

```diff
@@ -21,6 +21,8 @@
     "clean": "polkadot-dev-clean-build",
     "demo:identicon:react": "yarn build:ts && webpack-serve --config packages/react-identicon/webpack.config.js --content packages/react-identicon --port 8080",
     "demo:identicon:vue": "yarn build:ts && webpack-serve --config packages/vue-identicon/webpack.config.js --content packages/vue-identicon --port 8080",
+    "example:rn:ios": "yarn build:split:code && cd packages/exampleReactNative && yarn copy-workspace-packages && cd ios && pod install && cd .. && yarn ios && yarn start",
```

Comment:
Can we make this `example-reactnative` (I know it is a bit horrible, but it first in with the existing conventions)

Reply:
Do you mean rename the package to example-reactnative instead of exampleReactNative? I was planning on doing that, but react-native doesn't allow dashes in the project name.

Reply:
This is horrible - https://github.com/facebook/react-native/issues/213